Makefile

```diff
@@ -1,14 +1,12 @@
-start: restore backup
+test: restore backup
 
-restore: db
-	docker compose up -d restore
+restore:
+	docker compose run --rm app bash -c "s3cmd put /root/world* s3://world"
```

Comment:
local.env had S3_BUCKET and this is a hardcoded s3://world which is the bucket you created for testing. You can escape the environment variable so it gets glob'd from the container and not the host:
"s3cmd put /root/world* s3://\$S3_BUCKET"

Reply:
I don't know what you mean "from the container and not the host". Since documentation requires that the variable include the "s3://" part, we shouldn't include that here.

Reply:
> Also, using $S3_BUCKET in the Makefile would require you to define the variable in your host environment.

If you put \\$S3_BUCKET it is defined in the container not the host. I think some formatting ruined my example.

Reply:
Makes sense

Reply:
I found that the `$` must be doubled up for Makefile escape.